From: Jakub Jirutka <jakub@jirutka.cz>
Date: Thu, 7 Apr 2016 21:06:00 +0200
Subject: [PATCH] Do not bundle native binaries into JAR

The native library will be installed into /usr/lib as all other
platform-specific libraries.
---
diff --git a/build.xml b/build.xml
index 841582e..901b880 100644
--- a/build.xml
+++ b/build.xml
@@ -280,11 +280,9 @@
     <mkdir dir="${dist}" />
     <jar
       destfile="${dist}/${ivy.module}.jar">
-      <fileset dir="${src}/resources" erroronmissingdir="false" />
       <fileset dir="${build}/classes" />
       <fileset dir="${build}/unsafe-classes" />
       <fileset dir="${build}/generated-classes" />
-      <fileset dir="${build}/jni" erroronmissingdir="false" />
     </jar>
   </target>
 
